admin-console/src/main/webapp/src/components/EventTypes/EventTypeTable.tsx

```diff
@@ -101,7 +103,8 @@ const EventTypeTableImpl: React.FunctionComponent<EventTypeTableImplProps> = pro
                     />
                     <Td>{ eventType.displayName }</Td>
                     <Td>{ eventType.name }</Td>
-                    <Td><InstantEmailCell eventType={ eventType } /></Td>
+                    <Td><InstantEmailCell eventType={ eventType }
+                        onClick={ () => props.onCreateEditInstantTemplate } /></Td>
```

Comment:
combined here.

still can not get the modal to open. It is currently in the App page component. Should it live in this component?

Reply:
Not required - I'll try to pull the code tomorrow and see what part is missing.

Reply:
This part is never executed, should be:
```suggestion
                        onClick={ () => props.onCreateEditInstantTemplate(eventType.instantEmail.id ? eventType.instantEmail : undefined ) } /></Td>
```

Reply:
im unable to access the id on instantEmail... its only letting me use isLoading.

Reply:
any ideas here?